data/tutorials/getting-started/02_fst_program.md

```diff
@@ -0,0 +1,308 @@
+
+However, it is common practice to single out a value that triggers all the side effects and mark it as the intended main entry point. In OCaml, that's the role of the `let () =` pattern, which stands for “definition without a name” and actually indicates we're only interested in the side effects of the value on the right.
+
+## Modules and the Standard Library, Cont'd
+
+Let's summarise what was said about modules in the previous tutorial:
+- Modules are bundles of named values 
+- Identical names from distinct modules don't clash
+- The standard library is a bunch of modules
+
+In addition to that, modules are also the mean of organising projects. Concerns are separated into isolated modules. This is outlined in the next section. Before that, let's see how to use a definition from a module in our project: 
+```shell
+$ sed -i 's/print_endline/Printf.printf "%s!\\n"/g' bin/main.ml
+```
+
+This replaces the function `print_endline` with the function `printf` from the `Printf` module in the standard library. Building and executing this modified version should produce almost the same output as before. Use `dune exec hello` to try it for yourself.
+
+## Every File Defines a Module
+
+Most importantly, each OCaml file defines a module, once compiled. This is how separate compilation works in OCaml. Each sufficiently standalone concern should be isolated into a file-based module. References to external modules create dependencies. Circular dependencies between modules are not allowed.
+
+To create a module, let's create a file named `lib/hello.ml`:
+```shell
+$ echo 'let world = "Hello from a module"' > lib/hello.ml
+```
+
+Here is a new version of the `bin/main.ml` file and execution of the resulting project:
+```shell
+$ echo 'let () = Printf.printf "%s\n" Hello.world' > bin/main.ml
+
+$ dune exec hello
+Entering directory '/home/cuihtlauac/caml/ocaml.org'
+Hello from a module
+```
+
+The file `lib/hello.ml` contains the module named `Hello`, which in turn defines a string named `world`. This definition is referred to as `Hello.world` from the `bin/main.ml` file.
+
+Dune can launch UTop to access to the modules exposed by a project interactively. Here's how:
+```shell 
+$ dune utop
+```
+
+Then, inside the `utop` toplevel, it is possible to inspect our `Hello` module, just like we did when we examined the `Option` module in the previous tutorial.
+```ocaml
+# #show Hello;;
+module Hello : sig val world : string end
+```
+
+## Defining Module Interfaces
+
+What UTop's `#show` command displays is an API, the list of definitons provided by a module, which is called a _module interface_. In a similar way, module can be defined by `.ml` files. It is also possible to define module interfaces from files. Such files have a `.mli` extension and must share the same name part. 
```

Comment:
Two questions here:
"module can be defined by `.ml` files"
Do you mean "a module" or the word "module" here?

"must share the same name part" -- I'm unclear on what you mean by "name part" in this phrase.